remix/src/locales/en/translation.json

```
@@ -0,0 +1,5 @@
+{
+  "Page not found ;(": "Page not found ;(",
+  "You might have navigated to an unknown or an old page": "You might have navigated to an unknown or an old page",
+  "Home page": "Homepage"
+}
```

```suggestion
{
  "no-found": "Page not found ;(",
  "unkown-page": "You might have navigated to an unknown or an old page",
  "home-page": "Homepage"
}
```
Je pense que ce serait mieux de suivre les bonnes convention pour ecrire du json

---

Le problème de faire comme ça, c'est que la clé de ton JSON va chercher la string que tu lui donne pour la remplacer par la string de ta valeur.

Donc ça reviendrais à, dans ton composant, renommer la string "You might have navigated to an unknown or an old page" en "unkown-page". On fait quand même comme ça ?